utils/build.ps1

```diff
@@ -2293,11 +2293,14 @@ function Write-SDKSettingsPlist([OS] $OS) {
   Write-PList -Settings $SDKSettings -Path "$(Get-SwiftSDK $OS)\SDKSettings.plist"
 
   $SDKSettings = @{
-    CanonicalName = $OS.ToString()
-    DisplayName = $OS.ToString()
```

Comment:
What's the before/after value here?

Reply:
That should be `$OS.ToString()` still, but now we lowercase them properly.